cpp/src/arrow/ipc/options.h

```diff
@@ -67,6 +68,21 @@ struct ARROW_EXPORT IpcWriteOptions {
   /// May only be UNCOMPRESSED, LZ4_FRAME and ZSTD.
   std::shared_ptr<util::Codec> codec;
 
+  /// @brief Minimum space savings percentage required for compression to be applied
```

Comment:
nit: Arrow mostly uses `\brief` not Javadoc-style `@brief`

```suggestion
  /// \brief Minimum space savings percentage required for compression to be applied
```